node-module-system/add.js

```diff
@@ -0,0 +1,6 @@
+function add(num1, num2) {
+  const sum = num1 + num2;
+  return sum;
```

Comment:
This syntax works, of course, but most experienced developers would just write `return num1 + num2;`.